pyproject.toml

```diff
@@ -40,7 +40,8 @@ classifiers = [
 dependencies = [
     "oommfc>=0.65.0",
     "numpy>=1.19",
-    "scipy>=1.6"
+    "scipy>=1.6",
+    "hvplot"
```

Comment:
Should not be a dependency of mag2exp. It is only meant for the user so I would consider removing it. If you think installing it is important because it is otherwise too difficult for users, I would consider moving it to the metapackage like we do with jupyter lab.